src/main/xerus/monstercat/api/Player.kt

```diff
@@ -114,7 +114,11 @@ object Player: FadingHBox(true, targetHeight = 25) {
 	fun reset() {
 		fadeOut()
 		if(!Files.isDirectory(Settings.PLAYEREXPORTFILE())) {
-			Files.write(Settings.PLAYEREXPORTFILE(), arrayListOf(""), StandardOpenOption.CREATE)
+			Files.newBufferedWriter(Settings.PLAYEREXPORTFILE(),
```

Comment:
There should be a better way to clear it. Why `StandardOpenOption.CREATE`? Why not simply delete it?

Reply:
Deleting it and then trying to recreating it causes security exception on Windows

Reply:
Fixed in 9a4e5f5f81bf88c66e3fce8ea1b4ac5fd50152a2